torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -675,6 +675,19 @@ graph(%a_quant, %r_scale, %r_zero_point, %r_dtype):
          %r_quant = quantized::hardswish(%a_quant, %r_scale, %r_zero_point)
          return (%r_quant) )";
 
+  // quantized::elu
+  std::string elu = R"(
+graph(%a_quant, %alpha, %scale, %input_scale, %r_scale, %r_zero_point, %r_dtype):
+         %a_dequant = aten::dequantize(%a_quant)
+         %r = aten::elu(%a_dequant, %alpha, %scale, %input_scale)
+         %r_quant = aten::quantize_per_tensor(%r, %r_scale, %r_zero_point, %r_dtype)
+         return (%r_quant) )";
+
+  std::string quantized_elu = R"(
+graph(%a_quant, %alpha, %scale, %input_scale, %r_scale, %r_zero_point, %r_dtype):
+         %r_quant = quantized::elu(%a_quant, %r_scale, %r_zero_point, %alpha, %scale, %input_scale)
+         return (%r_quant) )";
```

Comment:
wait, where is the helper function for these things?
also we can quantize `aten::elu_` as well?

Reply:
yeah, the complication here is that this function has some optional arguments (the confusingly names "scale" and "input_scale" which have default values), so the quantization parameters are not the last args to this function.

TBH I don't think it's worth it to make the util function handle this case unless we have a lot more callsites - it's easier and more readable to just inline it.

Reply:
> aten::elu_

we removed the inplace version (in the first PR in this stack)

Reply:
OK, I think `elu_` can be quantized as well?

Reply:
I think it's fine to quantize `aten::elu_` to `quantized::elu`

Reply:
ah, I see.  sure.